cloud/linode/cloud.go

```diff
@@ -32,9 +33,10 @@ var supportedLoadBalancerTypes = []string{ciliumLBType, nodeBalancerLBType}
 // We expect it to be initialized with flags external to this package, likely in
 // main.go
 var Options struct {
-	KubeconfigFlag        *pflag.Flag
-	LinodeGoDebug         bool
-	EnableRouteController bool
+	KubeconfigFlag           *pflag.Flag
+	LinodeGoDebug            bool
+	EnableRouteController    bool
+	EnableTokenHealthChecker bool
```

Comment:
This flag should also be added to helm chart:
https://github.com/linode/linode-cloud-controller-manager/blob/main/deploy/chart/templates/daemonset.yaml
https://github.com/linode/linode-cloud-controller-manager/blob/main/deploy/chart/values.yaml